src/swap/handle_swap.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
+bool swap_check_validity();
```

Comment:
Not sure this file is necessary, we can put it in the src/swap/handle_swap_sign_transaction.c